crates/ruff/src/rules/perflint/snapshots/ruff__rules__perflint__tests__W8102_W8102.py.snap

```diff
@@ -0,0 +1,132 @@
+---
+source: crates/ruff/src/rules/perflint/mod.rs
+---
+W8102.py:7:5: W8102 [*] When using only the values of a dict use the `values()` method
+   |
+ 7 | }
+ 8 | 
+ 9 | for _, value in some_dict.items():  # W8102
+   |     ^ W8102
```

Comment:
I would say the diagnostics should be at `.items()` instead, the `_` is the reason due to which the violation happens. And we're also suggesting to replace `.items()`. What do you think?